src/Main.java

```diff
@@ -0,0 +1,113 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList<CoworkingSpace> spaces = new ArrayList<>();
+        ArrayList<Reservation> reservations = new ArrayList<>();
+
+        Admin admin = new Admin(spaces);
+        Customer customer = new Customer(reservations);
+
+        Scanner scanner = new Scanner(System.in);
+        int choice;
+
+
+        do {
+            System.out.println("""
+                    \n--- Welcome to Azamat's Coworking space ---
+                    --------------Main Menu----------------------
```

Comment:
try to use java text blocks: https://www.baeldung.com/java-text-blocks

Reply:
solved